libs/core/src/components/pds-button/pds-button.scss

```diff
@@ -1,5 +1,12 @@
 :host {
-  --button-dimension: 40px;
+  --pds-button-background: var(--color-background-default);
+  --pds-button-border: var(--pine-border);
+  --pds-button-border-radius: var(--pine-border-radius-full);
+  --pds-button-border-radius-start-end: var(--pine-border-radius-full);
+  --pds-button-border-radius-start-start: var(--pine-border-radius-full);
+  --pds-button-border-radius-end-end: var(--pine-border-radius-full);
+  --pds-button-border-radius-end-start: var(--pine-border-radius-full);
+  --pds-button-min-height: 40px;
```

Comment:
Do we need to define these here? If so, do we want consumers of this component to overwrite them?

Reply:
This was added so that buttons that exist within the slot of an input can receive appropriate styling. Outside of creating these in the host, I don't have a way to update the `border-radius-*` or other props of that button